docs/pages/reference/backends.mdx

```diff
@@ -270,6 +270,13 @@ To configure Teleport to use PostgreSQL:
   setting populated with the address of a load balancer for Auth Service
   instances.
 
+<Admonition
+  type="warning"
+  title="IMPORTANT"
```

Comment:
```suggestion
  title="PgBouncer"
```

The fact that this shows up as a warning is already indicating that it is important. Better to use a title that describes the content over a generic one.